config/printer-XYZ Printing-Davinci-2023.cfg

```diff
@@ -0,0 +1,232 @@
+rotation_distance:1.25
+endstop_pin: PB22
+position_endstop: -1
+position_max: 200
+position_min: -11
+
+
+#Extruder
+[extruder]
+step_pin: PB7
+dir_pin: PB6
+enable_pin: !PB8
+microsteps: 16
+rotation_distance: 32
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+max_extrude_only_distance: 110
+heater_pin: PA13
+sensor_pin: PB20 
+sensor_type: DaVinci_1
+control: pid
+pid_kp: 15.572
+pid_ki: 0.446
+pid_kd: 136.064
+min_temp: -273
+max_temp: 500
+
+#ADC values for extruder
+[adc_temperature DaVinci_1]
+temperature1: 25
+voltage1: 4.84661
+temperature2: 32
+voltage2: 4.69936
+temperature3: 64
+voltage3: 4.4846
+temperature4: 190
+voltage4: 2.42268
+temperature4: 220
+voltage4: 1.80307
+temperature4: 255
+voltage4: 1.70402
+#// extruder1; uncomment for second extruder for Duo Machines
+#[extruder1]
+#step_pin: PB14
+#dir_pin: PC28
+#enable_pin: !PB9
+#microsteps: 16
+#rotation_distance: 33.500
+#nozzle_diameter: 0.400
+#filament_diameter: 1.750
+#heater_pin: PB12
+#sensor_type: DaVinci_2
+#sensor_pin: PB18
+#control: pid
+#id_Kp: 22.2
+#pid_Ki: 1.08
+#pid_Kd: 114
+#min_temp: 0
+#max_temp: 250
+
+#// ADC values for extruder1; uncomment for second extruder for Duo Machines
+#[adc_temperature DaVinci_2]
+#temperature1: 25
+#voltage1: 4.84661
+#temperature2: 32
+#voltage2: 4.69936
+#temperature3: 64
+#voltage3: 4.4846
+#temperature4: 190
+#voltage4: 2.42268
+#temperature4: 220
+#voltage4: 1.80307
+#temperature4: 255
+#voltage4: 1.70402
+
+
+#uncomment for second extruder for Duo machines
+#[heater_fan my_nozzle_fan_1]
+#pin: PC26
+#heater_temp: 50.0
+#heater: extruder1
+
+[heater_fan my_nozzle_fan]
+pin: PD0
+heater: extruder
+heater_temp: 50.0
+fan_speed: 1.0
+
+
+
+[heater_bed]
+heater_pin: PB16
+sensor_type: DaVinci_3
+sensor_pin: PB21
+#control: watermark
+#max_delta: 2.0
+#pid_Kp: 27.108
+#pid_Ki: 0.421
+#pid_Kd: 436.094
+min_temp: 0
+max_temp: 110
+
+#ADC Values for heater_bed
+[adc_temperature DaVinci_3]
+temperature1: 25
+voltage1: 4.44661
+temperature2: 32
+voltage2: 4.29936
+temperature3: 64
+voltage3: 4.0846
+temperature4: 190
+voltage4: 2.02268
+temperature4: 220
+voltage4: 1.30307
+temperature4: 255
+voltage4: 1.30402
+
+
+[bed_screws]
+screw1: 75,50
+screw1_name: rear center center
+screw2: 125,175
+screw2_name:front left screw
+screw3: 25,175
+screw3_name:front right screw
+
+
+[printer]
+kinematics: cartesian
+max_velocity: 100
+max_accel: 500
+max_z_velocity: 5
+max_z_accel: 10
+
+#LED lighting
+# Chamber Lighting
+[output_pin caselight]
+pin: PA12
+#max_power: 1.0
+#kick_start_time: 0.5
+pwm: true
+shutdown_value: 0
+value: 100
+cycle_time: 0.01
+scale: 100
+ 
+# Lighting Control
+[gcode_macro lights_off]
+gcode:
+    SET_PIN PIN=caselight VALUE=0
+ 
+[gcode_macro lights_on]
+gcode:
+    SET_PIN PIN=caselight VALUE=100
+ 
+[gcode_macro lights_toggle]
+gcode:
+    {% if printer["output_pin caselight"].value == 100 %}
+        lights_off
+    {% else %}
+        lights_on
+    {% endif %}
+
+
+
+#expiremental - not tested

```

Comment:
pins are active and function as expected. inductive style probe and must contact metal pads on the edge of bed to function.